Source/DafnyLanguageServer/DafnyLanguageServer.cs

```diff
@@ -35,6 +36,15 @@ public static class DafnyLanguageServer {
         Action killLanguageServer) {
       var logger = server.GetRequiredService<ILogger<Program>>();
       logger.LogTrace("initializing service");
+      var dafnyPluginsOptions = server.GetRequiredService<IOptions<DafnyPluginsOptions>>();
```

Comment:
I'd prefer having the plugin load code being extracted into a dedicated method.
One usual pattern I apply: If I start grouping statements with empty lines it's a good sign to extract the code into a new method.